### ISSUE TITLE
Auto detect if `imwrite` should use ome and bigtiff or not

### ISSUE THREAD

**Issue by matt-sd-watson** [BodenmillerGroup/steinbock#275]
Related to: https://github.com/cgohlke/tifffile/issues/279

Recently, we have collaborators that report truncation of tifffiles written at the `steinbock preprocess imc images` step [here](https://github.com/BodenmillerGroup/steinbock/issues/271), presuemably because of the ImageJ 4GB size limit. 

I believe that this could be resolved if the pipeline were able to detect when a sufficiently large array should be written as big tiff or not. It appears that this option is not used currently: https://github.com/BodenmillerGroup/steinbock/blob/7b73f2925b68a351925e89aa145a7a739741b2b7/steinbock/io.py#L191

**Comment by jwindhager** [comment]
Indeed, steinbock writes ImageJ-compliant TIFF files (i.e., max 4GB). Given the recent hardware developments, I'd endorse this suggestion.

**Comment by jwindhager** [comment]
Likely related to https://github.com/BodenmillerGroup/steinbock/issues/271
